packages/fluentui/react-northstar/src/components/Datepicker/DatepickerCalendarCell.tsx

```diff
@@ -66,7 +75,17 @@ export const DatepickerCalendarCell = compose<
     const { setStart, setEnd } = useTelemetry(composeOptions.displayName, context.telemetry);
     setStart();
 
-    const { className, design, styles, variables, disabled, selected, content } = props;
+    const {
+      className,
+      design,
+      styles,
+      variables,
+      disabled,
+      selected,
+      quiet: unfocused,
```

Comment:
```suggestion
      quiet,
```
Let's get rid of `unfocused` and `reference` altogether

Reply:
Oh sorry about this (facepalm) I was naively relying on auto-rename functionality in the refactoring suite.